test/constants.js

```diff
@@ -66,6 +66,7 @@ const Path = {
   HIP583_ERC721Mock: 'contracts/hip-583/ERC721Mock.sol:ERC721Mock',
   HRC: 'contracts/hrc/HRC.sol:HRC',
   TYPE_OPS: 'contracts/solidity/typeops/TypeOps.sol:TypeOps',
+  RECEIVER_PAYS: 'contracts/solidity/signature-example/ReceiverPays.sol:ReceiverPays',
```

Comment:
Q: Should we simplify it to signature?

Reply:
I based myself on the names of the contracts described on the example provided with the issue.
https://docs.soliditylang.org/en/latest/solidity-by-example.html#creating-and-verifying-signatures 

But I can make the refactor nonetheless. 👍